subprojects/internal-performance-testing/src/main/resources/org/gradle/reporting/performanceGraph.js

```diff
@@ -1,113 +1,137 @@
 (function ($) {
-  var renderCommitIds = function(commits) {
-    return commits.map(function(commit) {
-        return commit.substring(0, 7);
-    }).join('|');
-  }
+    var renderCommitIds = function(commits) {
```

Comment:
Previously the indentations are mixed by 2 spaces and 4 spaces. I think it's better to reformat it with our checked-in code style configuration.